### PR TITLE
please add openVALIDATION Project to readme

### PR DESCRIPTION
**Pull request by thecodemonkey**
please add openVALIDATION Project to the readme.

This open source project includes the integration of openVALIDATION into OpenAPI. It enables the specification of complex validation rules within a service contract using natural language. Thus, both the service stubs and the client proxies including validation logic can be generated automatically. [ read more...](https://docs.openvalidation.io/openapi/openapi-specification)

image

validation rules can also be generated separately.

sample:

```yaml
openapi: "3.0.0"
info:
  version: 1.0.0
  title: sample
paths:
  /:
    post:
      requestBody:
        content:
          application/json:
            schema:
              properties:
                name:
                  type: string
                mail:
                  type: number
            x-ov-rules:
              culture: en
              rule: |  
                the name of applicant has to be Alex
                and his mail must not be alex@yahoo.com
      responses:
        '200':
          description: ok
```

### PR checklist

- [ x ] Read the [contribution guidelines](https://github.com/openapitools/openapi-generator/blob/master/CONTRIBUTING.md).
- [ x ] If contributing template-only or documentation-only changes which will change sample output, [build the project](https://github.com/OpenAPITools/openapi-generator#14---build-projects) before.

### CODE REVIEW

**Comment by spacether** [comment]
Looking through our source code, I don't see any examples or code that processes or include `x-ov-rules`. Isn't this a completely separate tool?

**Comment by thecodemonkey** [comment]
openVALIDATION is the core project. The integration into OpenAPI is a subproject called openVALIDATION-OpenAPI:

https://github.com/openvalidation/openvalidation-openapi

here is the resolution of the **x-ov-rules** extension

https://github.com/openvalidation/openvalidation-openapi/search?q=x-ov-rules&unscoped_q=x-ov-rules

here are the corresponding docs:

https://docs.openvalidation.io/openapi/openapi-specification

File: README.md [hunk 1]
```diff
@@ -615,6 +615,7 @@ Here are some companies/projects (alphabetical order) using OpenAPI Generator in
 - [Myworkout](https://myworkout.com)
 - [NamSor](https://www.namsor.com/)
 - [Openet](https://www.openet.com/)
+- [openVALIDATION] (https://openvalidation.io/)
```
Comment (ackintosh) [inline README.md]: Please remove the space. 😃 

```suggestion
- [openVALIDATION](https://openvalidation.io/)
```

Preview: 

image

**Comment by wing328** [comment]
@thecodemonkey thanks for the PR, which has been included in the v4.3.1 release: https://twitter.com/oas_generator/status/1258057660974329861

**Comment by thecodemonkey** [comment]
> @thecodemonkey thanks for the PR, which has been included in the v4.3.1 release: https://twitter.com/oas_generator/status/1258057660974329861


Thanks for accepting the PR 👍😀

**Pull request merged**